pyproject.toml

```
@@ -80,3 +80,6 @@ convention = "numpy"  # Accepts: "google", "numpy", or "pep257".
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.semantic_release]
+version_variables = ["pyproject.toml:version"]
```

Add missing newline at the end of the file.
```suggestion
version_variables = ["pyproject.toml:version"]

```